apps/server/lib/bootstrap.ts

```diff
@@ -302,6 +397,73 @@ export const bootstrap = async (logContext: LogContext, options: any) => {
 		}),
 	);
 
+	// Manually sync oauth clients
+	// TODO: This should ideally be completely automated, but this would
+	// require that triggered actions are up and running before any
+	// oauth-client contracts are loaded.
+	const jellyfishOauthProvider = await kernel.replaceContract(
+		logContext,
+		kernel.adminSession()!,
+		{
+			slug: 'oauth-provider-jellyfish',
+			type: 'oauth-provider@1.0.0',
+			name: 'Jellyfish oauth provider',
+			data: {
+				authorizeUrl: `${environment.hydra.publicBaseUrl}/oauth2/auth?client_id={{clientId}}&response_type=code&state={{state}}`,
+				tokenUrl: `${environment.hydra.publicBaseUrl}/oauth2/token`,
+				integration: 'jellyfish',
+			},
+		},
+	);
+
+	const jellyfishOauthClient = await kernel.replaceContract(
+		logContext,
+		kernel.adminSession()!,
+		{
+			slug: 'oauth-client-jellyfish',
+			type: 'oauth-client@1.0.0',
+			name: 'Jellyfish oauth client',
+			data: {
+				clientId: environment.integration.jellyfish.appId,
+				clientSecret: environment.integration.jellyfish.appSecret,
+				redirectUrl: `${environment.livechat.baseUrl}/oauth/callback`,
+			},
+		},
+	);
+
+	await kernel.replaceContract(logContext, kernel.adminSession()!, {
+		type: 'link@1.0.0',
+		name: 'has attached',
+		slug: `link-${jellyfishOauthProvider!.id}-has-attached-${
+			jellyfishOauthClient.id
+		}`,
+		data: {
+			inverseName: 'is attached to',
+			from: {
+				id: jellyfishOauthProvider!.id,
+				type: jellyfishOauthProvider!.type,
+			},
+			to: {
+				id: jellyfishOauthClient.id,
+				type: jellyfishOauthClient.type,
+			},
+		},
+	});
```

Comment:
Need to preinsert and link not only jellyfish oauth contracts, but also outreach and balena api.